tests/helper_networks.rs

```diff
@@ -0,0 +1,97 @@
+use std::{
+    error::Error,
+    io::{self, Write},
+    iter::zip,
+    process::{Command, ExitStatus, Stdio},
+    str,
+};
+use tempfile::tempdir;
+
+const HELPER_BIN: &str = env!("CARGO_BIN_EXE_helper");
+const TEST_MPC_BIN: &str = env!("CARGO_BIN_EXE_test_mpc");
```

Comment:
Unfortunately, I don't have a good alternative for this.  I assume that we can't import code from those modules so that we could statically enforce correctness, like:

```rust
use crate::helper;
const HELPER_BIN: &str = env!(concat!("CARGO_BIN_EXE_", stringify!(helper)));
```

(I'm not even sure if that would catch a rename or move of helper...)

Reply:
Yeah, I'm not sure how much better we can do here assuming we want to invoke the binaries. If we structured the tools as functions exposed by the `ipa` crate with trivial `main` wrappers, then we could invoke the functions from this test.

In the event one of these gets renamed, failure to update this shouldn't escape CI -- it's just something that would be caught by tests rather than build.